dev/AppNotifications/AppNotificationBuilder/AppNotificationComboBox.cpp

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "AppNotificationComboBox.h"
+#include "Microsoft.Windows.AppNotifications.Builder.AppNotificationComboBox.g.cpp"
+#include "AppNotificationBuilderUtility.h"
+
+namespace winrt::Microsoft::Windows::AppNotifications::Builder::implementation
+{
+    AppNotificationComboBox::AppNotificationComboBox(hstring const& id) : m_id(id) { };
+
+    winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationComboBox AppNotificationComboBox::AddItem(winrt::hstring const& id, winrt::hstring const& content)
+    {
+        THROW_HR_IF_MSG(E_INVALIDARG, m_items.Size() >= c_maxSelectionElements, "Maximum number of items added");
+        THROW_HR_IF_MSG(E_INVALIDARG, id.empty(), "You must provide an id when adding an argument");
+
+        m_items.Insert(id, content);
+
+        return *this;
+    }
+
+    winrt::Microsoft::Windows::AppNotifications::Builder::AppNotificationComboBox AppNotificationComboBox::SetTitle(winrt::hstring const& value)
+    {
+        m_title = value;
```

Comment:
Please make sure that value isn't empty here

Reply:
Captured in this deliverable https://microsoft.visualstudio.com/OS/_workitems/edit/40746502.